backend/czi_hosted/data_common/matrix_loader.py

```diff
@@ -152,6 +155,21 @@ def data_adaptor(self, url_dataroot, location, app_config):
                 cache_item = info.cache_item
 
             if data_adaptor is None:
+                if app_config.server_config.data_locator__api_base:
+                    if url_dataroot == "e":  # only pull dataset identification information from the portal for /e/ route
```

Comment:
this seems like a fragile assumption (the 'e' value).  Is it really fixed/always true?  Or should it be a config param?

Reply:
We only want to look it up for the `/e/` route because that is the route for datasets in the data portal. Other dataset paths wont be in the portal/should default to the name based lookup used historically.

Reply:
`/e/` is the current route on the portal, but that could change.  Just confirming that the intent here is to require a code change to the Explorer backend if the portal changes their routes (rather than adding a configuration parameter to Explorer, ie, `PortalDatasetBaseURL: "/e/"`)

I don't feel super strongly, but my point was that hard-coding routes like this is brittle - it forces a release (code change) to Explorer whenever Portal changes it's configuration.